### PR TITLE
【Docs】Add ipynb

### PR DESCRIPTION
**Pull request by VVsssssk**
## Motivation

Add colab ipynb file.

## Modification

demo/tutorials_1.ipynb

### CODE REVIEW

**Comment by RunningLeon** [comment]
@VVsssssk Could you fix lint error?

**Comment by VVsssssk** [comment]
2022-03-15 16:21:21,723 - mmdeploy - INFO - **********Environmental information**********
2022-03-15 16:21:22,543 - mmdeploy - INFO - sys.platform: linux
2022-03-15 16:21:22,543 - mmdeploy - INFO - Python: 3.8.5 (default, Sep  4 2020, 07:30:14) [GCC 7.3.0]
2022-03-15 16:21:22,543 - mmdeploy - INFO - CUDA available: True
2022-03-15 16:21:22,543 - mmdeploy - INFO - GPU 0: GeForce GTX 1660 SUPER
2022-03-15 16:21:22,543 - mmdeploy - INFO - CUDA_HOME: /usr/local/cuda-11.1
2022-03-15 16:21:22,543 - mmdeploy - INFO - NVCC: Build cuda_11.1.TC455_06.29190527_0
2022-03-15 16:21:22,543 - mmdeploy - INFO - GCC: gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
2022-03-15 16:21:22,543 - mmdeploy - INFO - PyTorch: 1.8.2+cu111
2022-03-15 16:21:22,543 - mmdeploy - INFO - PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2020.0.0 Product Build 20191122 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v1.7.0 (Git Hash 7aed236906b1f7a05c0917e5257a1af05e9ff683)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 11.1
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_86,code=sm_86
  - CuDNN 8.0.5
  - Magma 2.5.2
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.1, CUDNN_VERSION=8.0.5, CXX_COMPILER=/opt/rh/devtoolset-7/root/usr/bin/c++, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_KINETO -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_VERSION=1.8.2, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, 

2022-03-15 16:21:22,543 - mmdeploy - INFO - TorchVision: 0.9.2+cu111
2022-03-15 16:21:22,543 - mmdeploy - INFO - OpenCV: 4.5.5
2022-03-15 16:21:22,543 - mmdeploy - INFO - MMCV: 1.4.6
2022-03-15 16:21:22,543 - mmdeploy - INFO - MMCV Compiler: GCC 7.3
2022-03-15 16:21:22,543 - mmdeploy - INFO - MMCV CUDA Compiler: 11.1
2022-03-15 16:21:22,543 - mmdeploy - INFO - MMDeploy: 0.4.0+00ca2a3
2022-03-15 16:21:22,544 - mmdeploy - INFO - 

2022-03-15 16:21:22,544 - mmdeploy - INFO - **********Backend information**********
2022-03-15 16:21:22,701 - mmdeploy - INFO - onnxruntime: 1.8.1	ops_is_avaliable : True
2022-03-15 16:21:22,701 - mmdeploy - INFO - tensorrt: 8.0.1.6	ops_is_avaliable : True
2022-03-15 16:21:22,701 - mmdeploy - INFO - ncnn: 1.0.20220309	ops_is_avaliable : False
2022-03-15 16:21:22,702 - mmdeploy - INFO - pplnn_is_avaliable: True
2022-03-15 16:21:22,703 - mmdeploy - INFO - openvino_is_avaliable: True
2022-03-15 16:21:22,703 - mmdeploy - INFO - 

2022-03-15 16:21:22,703 - mmdeploy - INFO - **********Codebase information**********
2022-03-15 16:21:22,705 - mmdeploy - INFO - mmdet:	2.20.0
2022-03-15 16:21:22,705 - mmdeploy - INFO - mmseg:	0.20.2
2022-03-15 16:21:22,705 - mmdeploy - INFO - mmcls:	0.16.0
2022-03-15 16:21:22,705 - mmdeploy - INFO - mmocr:	None
2022-03-15 16:21:22,705 - mmdeploy - INFO - mmedit:	0.11.0
2022-03-15 16:21:22,705 - mmdeploy - INFO - mmdet3d:	1.0.0rc0
2022-03-15 16:21:22,705 - mmdeploy - INFO - mmpose:	0.22.0

**Comment by VVsssssk** [comment]
image
image

**Comment by AllentDan** [comment]
Do we have the English versions and how do we organize them then?

File: demo/tutorials_1.ipynb [hunk 1]
```diff
@@ -0,0 +1,484 @@
...
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# 安装 cuda 11.1 的 PyTorch \n",
+        "# 如果你用的是其他版本的 cuda，请参考 PyTorch 的官方安装教程选择安装命令 \n",
+        "!pip install torch==1.8.2+cu111 torchvision==0.9.2+cu111 torchaudio==0.8.2 -f https://download.pytorch.org/whl/lts/1.8/torch_lts.html\n",
```
Comment (RunningLeon) [inline demo/tutorials_1.ipynb:113]: remove torchaudio since it is not used in anywhere

File: demo/tutorials_1.ipynb [hunk 2]
```diff
@@ -0,0 +1,484 @@
...
+        " \n",
+        "# Download checkpoint and test image \n",
+        "urls = ['https://download.openmmlab.com/mmediting/restorers/srcnn/srcnn_x4k915_1x16_1000k_div2k_20200608-4186f232.pth', \n",
+        "    'https://raw.githubusercontent.com/open-mmlab/mmediting/master/tests/data/face/000001.png'] \n",
+        "names = ['srcnn.pth', 'face.png'] \n",
+        "for url, name in zip(urls, names): \n",
+        "    if not os.path.exists(name): \n",
+        "        open(name, 'wb').write(requests.get(url).content) \n",
+        " \n",
+        "def init_torch_model(): \n",
```
Comment (RunningLeon) [inline demo/tutorials_1.ipynb:235]: why there is always one empty space at the end of every line like `{ONE_SPACE}\n`

**Comment by RunningLeon** [comment]
@VVsssssk Hi, pls. update this PR and hope it be merged soon.

**Comment by VVsssssk** [comment]
> @VVsssssk Hi, pls. update this PR and hope it be merged soon.

OK

**Comment by SingleZombie** [comment]
Do we need to create a new folder 'tutorial' in the ./demo? The number of notebook files is increasing and they will make the ./demo untidy.

**Comment by RunningLeon** [comment]
> Do we need to create a new folder 'tutorial' in the ./demo? The number of notebook files is increasing and they will make the ./demo untidy.

@SingleZombie Agree.

**Comment by RunningLeon** [comment]
@VVsssssk Pls. fix lint.

**Review by RunningLeon — APPROVED** [review]
LGTM

**Review by AllentDan — APPROVED** [review]
LGTM

**Pull request merged**